src/Infrastructure.EntityFramework/Repositories/CipherRepository.cs

```diff
@@ -562,7 +562,8 @@ public async Task UpdateAttachmentAsync(CipherAttachment attachment)
             var attachments = string.IsNullOrWhiteSpace(cipher.Attachments) ?
                 new Dictionary<string, CipherAttachment.MetaData>() :
                 JsonConvert.DeserializeObject<Dictionary<string, CipherAttachment.MetaData>>(cipher.Attachments);
-            attachments.Add(attachment.AttachmentId, JsonConvert.DeserializeObject<CipherAttachment.MetaData>(attachment.AttachmentData));
+            var metaData = JsonConvert.DeserializeObject<CipherAttachment.MetaData>(attachment.AttachmentData);
+            attachments[attachment.AttachmentId] = metaData;
```

Comment:
Unrelated/unintended change?

Reply:
This must have been from a merge from master. I've undone this change.